conda_env/cli/main_create.py

```diff
@@ -81,6 +88,8 @@ def execute(args, parser):
         common.error_and_exit(str(e), json=args.json)
 
     prefix = common.get_prefix(args, search=False)
+    if args.force and prefix != root_env_name and os.path.exists(prefix):
+        shutil.rmtree(prefix)
```

Comment:
You can also benefit from `is_root_prefix` as in https://github.com/conda/conda-env/blob/develop/conda_env/cli/main_remove.py#L46